src/Campaigns/Blocks/shared/hooks/useCampaign.ts

```diff
@@ -2,10 +2,12 @@ import {useEntityRecord} from '@wordpress/core-data';
 import {Campaign} from '@givewp/campaigns/admin/components/types';
 
 export default function useCampaign(campaignId: number) {
-    const data = useEntityRecord('givewp', 'campaign', campaignId);
+    const campaignData = useEntityRecord('givewp', 'campaign', campaignId);
 
     return {
-        campaign: data?.record as Campaign,
-        hasResolved: data?.hasResolved,
+        campaign: {
+            ...campaignData?.record as Campaign
+        },
+        hasResolved: campaignData?.hasResolved,
```

Comment:
Do we still need these changes here?

Reply:
Nope. Resolved https://github.com/impress-org/givewp/pull/7687/commits/8725b512c554ec0cabecc8e1da4443173fcf5701